packages/cli/src/utils.ts

```diff
@@ -174,6 +167,29 @@ const checkImageExtension = (uri: string): boolean => {
   );
 };
 
+/**
+ * We need to pre-check some things in the localized resources before we move forward
+ */
+const validateLocalizableResources = (config: CLIConfig) => {
+  const baselineSize = Object.keys(config.release.catalog["en-US"]).length;
+  Object.keys(config.release.catalog).forEach((locale) => {
+    const size = Object.keys(config.release.catalog[locale]).length;
+
+    if (size != baselineSize) {
+      throw new Error("Please ensure you have included all localized strings for all locales in your configuration file.");
+    }
+  });
+
+  const descsTooLong = Object.values(config.release.catalog)
```

Comment:
MINOR: you could probably use the `every` method with an appropriate predicate here, but that's just an optimization; the check is fine as written.